// @primer/primitives/tokens-v2-private/js/tokens/functional/typography/typography.js
/**
 * Do not edit directly
 * Generated on Fri, 29 Apr 2022 19:38:32 GMT
 */

export const PrimerFontStackSystem = "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const PrimerFontStackSansSerif = "-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'";
export const PrimerFontStackMonospace = "ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace";
export const PrimerTextDisplayLineBoxHeight = "3.5rem";
export const PrimerTextDisplaySize = "2.5rem";
export const PrimerTextDisplayLineHeight = "3.5rem";
export const PrimerTextDisplayWeight = "500";
export const PrimerTextDisplayShorthand = {"fontWeight":"500","fontSize":"2.5rem","lineHeight":"3.5rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextTitleSizeLarge = "2rem";
export const PrimerTextTitleSizeMedium = "1.25rem";
export const PrimerTextTitleLineHeightLarge = "3rem";
export const PrimerTextTitleLineHeightMedium = "2rem";
export const PrimerTextTitleWeight = "600";
export const PrimerTextTitleShorthandLarge = {"fontWeight":"600","fontSize":"2rem","lineHeight":"3rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextTitleShorthandMedium = {"fontWeight":"600","fontSize":"1.25rem","lineHeight":"2rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextSubtitleSize = "1.25rem";
export const PrimerTextSubtitleLineHeight = "2rem";
export const PrimerTextSubtitleWeight = "400";
export const PrimerTextSubtitleShorthand = {"fontWeight":"400","fontSize":"1.25rem","lineHeight":"2rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodySizeLarge = "1rem";
export const PrimerTextBodySizeMedium = "0.875rem";
export const PrimerTextBodySizeSmall = "0.75rem";
export const PrimerTextBodyLineHeightLarge = "1.5rem";
export const PrimerTextBodyLineHeightMedium = "1.25rem";
export const PrimerTextBodyLineHeightSmall = "1.25rem";
export const PrimerTextBodyWeight = "400";
export const PrimerTextBodyShorthandLarge = {"fontWeight":"400","fontSize":"1rem","lineHeight":"1.5rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodyShorthandMedium = {"fontWeight":"400","fontSize":"0.875rem","lineHeight":"1.25rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextBodyShorthandSmall = {"fontWeight":"400","fontSize":"0.75rem","lineHeight":"1.25rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextCaptionSize = "0.75rem";
export const PrimerTextCaptionLineHeight = "1rem";
export const PrimerTextCaptionWeight = "400";
export const PrimerTextCaptionShorthand = {"fontWeight":"400","fontSize":"0.75rem","lineHeight":"1rem","fontFamily":"-apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji'"};
export const PrimerTextCodeBlockSize = "0.8125rem";
export const PrimerTextCodeBlockLineHeight = "1.25rem";
export const PrimerTextCodeBlockWeight = "400";
export const PrimerTextCodeBlockShorthand = {"fontWeight":"400","fontSize":"0.8125rem","lineHeight":"1.25rem","fontFamily":"ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace"};
export const PrimerTextCodeInlineSize = "0.9285em";
export const PrimerTextCodeInlineWeight = "400";
export const PrimerTextCodeInlineShorthand = {"fontWeight":"400","fontSize":"0.9285em","lineHeight":"inherit","fontFamily":"ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace"};
