// @primer/primitives/tokens-v2-private/js/tokens/functional/size/size.js
/**
 * Do not edit directly
 * Generated on Fri, 29 Apr 2022 19:38:32 GMT
 */

export const PrimerControlMinTargetFine = "1rem";
export const PrimerControlMinTargetCoarse = "2.75rem";
export const PrimerControlXsmallSize = "1.5rem";
export const PrimerControlXsmallLineBoxHeight = "1.25rem";
export const PrimerControlXsmallPaddingBlock = "calc((1.5rem - 1.25rem) / 2)";
export const PrimerControlXsmallPaddingInlineCondensed = "0.25rem";
export const PrimerControlXsmallPaddingInlineNormal = "0.5rem";
export const PrimerControlXsmallPaddingInlineSpacious = "0.75rem";
export const PrimerControlXsmallGap = "0.25rem";
export const PrimerControlSmallSize = "1.75rem";
export const PrimerControlSmallLineBoxHeight = "1.25rem";
export const PrimerControlSmallPaddingBlock = "calc((1.75rem - 1.25rem) / 2)";
export const PrimerControlSmallPaddingInlineCondensed = "0.5rem";
export const PrimerControlSmallPaddingInlineNormal = "0.75rem";
export const PrimerControlSmallGap = "0.25rem";
export const PrimerControlMediumSize = "2rem";
export const PrimerControlMediumLineBoxHeight = "1.25rem";
export const PrimerControlMediumPaddingBlock = "calc((2rem - 1.25rem) / 2)";
export const PrimerControlMediumPaddingInlineCondensed = "0.5rem";
export const PrimerControlMediumPaddingInlineNormal = "0.75rem";
export const PrimerControlMediumPaddingInlineSpacious = "1rem";
export const PrimerControlMediumGap = "0.5rem";
export const PrimerControlLargeSize = "2.5rem";
export const PrimerControlLargeLineBoxHeight = "1.25rem";
export const PrimerControlLargePaddingBlock = "calc((2.5rem - 1.25rem) / 2)";
export const PrimerControlLargePaddingInlineNormal = "0.75rem";
export const PrimerControlLargePaddingInlineSpacious = "1rem";
export const PrimerControlLargeGap = "0.5rem";
export const PrimerControlXlargeSize = "3rem";
export const PrimerControlXlargeLineBoxHeight = "1.25rem";
export const PrimerControlXlargePaddingBlock = "calc((3rem - 1.25rem) / 2)";
export const PrimerControlXlargePaddingInlineNormal = "0.75rem";
export const PrimerControlXlargePaddingInlineSpacious = "1rem";
export const PrimerControlXlargeGap = "0.5rem";
export const PrimerStackPaddingCondensed = "0.5rem";
export const PrimerStackPaddingNormal = "1rem";
export const PrimerStackPaddingSpacious = "1.5rem";
export const PrimerStackGapCondensed = "0.5rem";
export const PrimerStackGapNormal = "1rem";
export const PrimerStackGapSpacious = "1.5rem";
export const PrimerControlStackSmallGapCondensed = "0.5rem";
export const PrimerControlStackSmallGapSpacious = "1rem";
export const PrimerControlStackMediumGapCondensed = "0.5rem";
export const PrimerControlStackMediumGapSpacious = "0.75rem";
export const PrimerControlStackLargeGapAuto = "0.5rem";
export const PrimerControlStackLargeGapCondensed = "0.5rem";
export const PrimerControlStackLargeGapSpacious = "0.75rem";
